### PR TITLE
MM-T3080 Add Cypress test for Sign in email/pwd account

### PR DESCRIPTION
**Pull request by yulyanaR**
**Summary**

Cypress test for Sign in email/pwd account MM-T3080

**Ticket Link**

https://automation-test-cases.vercel.app/test/MM-T3080

### CODE REVIEW

File: e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js
Comment (Willyfrog) [inline e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js:11]: I believe this has to be filled by QA, so I'd remove them

**Review by Willyfrog — APPROVED** [review]
looking good minus one thing regarding how qa usually does this process.

**Comment by yulyanaR** [comment]
Fixed as per the request, waiting for approval. Thanks!

File: e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js
Comment (saturninoabril) [inline e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js:16]:
```suggestion
        // # Create new user
```

File: e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js
Comment (saturninoabril) [inline e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js:26]:
```suggestion
        // # Enter actual user's email in the email field
```

File: e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js
Comment (saturninoabril) [inline e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js:31]:
```suggestion
            // # Enter user's password in the password field
```

File: e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js
Comment (saturninoabril) [inline e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js:49]:
```suggestion
            // # Enter actual user's username in the email field
```

File: e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js
Comment (saturninoabril) [inline e2e/cypress/integration/signin_authentication/login_logout_smoke_spec.js:52]:
```suggestion
            // # Enter user's password in the password field
```

**Review by saturninoabril — APPROVED** [review]
@yulyanaR Thanks for updating! I left last comments which I missed to mention last time. Approving now given those are to be updated.

**Review by prapti — APPROVED** [review]
Hi @yulyanaR! This is great! The test flow is exactly what's needed and you did it perfectly. Thank you so much for your contribution and congratulations on your first Mattermost PR 🎉

**Pull request merged**
